bleu.py

```diff
@@ -0,0 +1,35 @@
+#wip
```

Comment:
Add a comment on what this file is about. Use triple quotes for this kind of comments.